### PR TITLE
compactify log header

### PR DESCRIPTION
**Pull request by tbg**
> L="ERROR" T="5/13 10:35:54.456" F="start.go:118" Msg="Whatever" OtherField="..." ...

### CODE REVIEW

File: util/log/log.go [hunk 1]
```diff
@@ -27,6 +27,8 @@ import (
 	"golang.org/x/net/context"
 )
 
+const timeFormat = "1/2 15:04:05.000" // m/d h:m:s.nanos
```
Comment (MBerhault) [inline util/log/log.go]: we should really have YYYY/MM/DD as the date format.
Actually, time.RFC3339 seemed reasonable.

Comment (tschottdorf) [inline util/log/log.go]: I should have mentioned that currently it's more important to have readable log output, and there have been complaints from the team about eye cancer. We'll have decent timestamps eventually, but possibly we'll also just have logs for humans and machines separately.

Comment (MBerhault) [inline util/log/log.go]: fair enough, but let's at least put the year in there.

**Comment by MBerhault** [comment]
LGTM

File: util/log/log.go (hunk 1 above)
Comment (tschottdorf) [inline util/log/log.go]: I left it out deliberately. The year seems the most redundant thing for a human running a local cluster.

Comment (bdarnell) [inline util/log/log.go]: I agree with leaving out the year (the old glog format didn't have the year either), but let's zero-pad the month and day so the header is fixed-width.

**Comment by bdarnell** [comment]
LGTM

File: util/log/log.go (hunk 1 above)
Comment (BramGruneir) [inline util/log/log.go]: I agree with Marc.  The year should be in there.  And we should zero-pad everything. 
yyyy-mm-dd hh:mm-ss.nnn 
The year makes it very clear the order of months and days.

Comment (tamird) [inline util/log/log.go]: +1 year adds clarity

**Comment by tschottdorf** [comment]
Save your energy guys. This is an interim solution for the next few weeks.

**Pull request merged**